src/components/partner/dto/partner.dto.ts

```diff
@@ -51,8 +51,13 @@ export class Partner extends Interfaces {
   @Field()
   readonly pmcEntityCode: SecuredStringNullable;
 
+  @Field({
+    deprecationReason: 'Use `growthPartnersClient` instead',
+  })
+  readonly globalInnovationsClient?: SecuredBoolean;
```

Comment:
This is not needed, because we have the field resolver that will map the new property name/value to the old one.
We don't want to declare that our internal shapes have this property - they have the new one instead.